src/pages/LandingPage/index.jsx

```diff
@@ -1,8 +1,11 @@
 import { Buttons } from "../../components/Buttons";
 import "./style.css";
 import finalHomepage from "../../assets/finalHomepage.png"
+import { useNavigate } from "react-router-dom";
 
 export const LandingPage = () => {
+  const navigateTo = useNavigate()
+
   return (
     <div className="landing-page">
       <div className="div">
```

Comment:
This navbar needs to be separated into a component, so that it can be used across all pages e.g the roadmap page. It also needs to be updated to include the log in/sign up buttons and log out button (for logged in users), as shown in the designs